packages/manifold/contracts/edition/ManifoldERC721Edition.sol

```diff
@@ -134,25 +128,27 @@ contract ManifoldERC721Edition is CreatorExtension, ICreatorExtensionTokenURI, I
             indexRanges.push(IndexRange(startIndex, count));
           }
         }
-        _totalSupply[creator][series] += count;
+        _totalSupply[creatorCore][instanceId] += count;
     }
 
     /**
      * @dev Index from tokenId
      */
-    function _tokenSeriesAndIndex(address creator, uint256 tokenId) internal view returns(uint256, uint256) {
-        require(_currentSeries[creator] > 0, "Invalid token");
-        for (uint series=1; series <= _currentSeries[creator]; series++) {
-            IndexRange[] memory indexRanges = _indexRanges[creator][series];
+    function _tokenInstanceAndIndex(address creatorCore, uint256 tokenId) internal view returns(uint256, uint256) {
+        // Go through all their series until we find the tokenId
+        for (uint256 i = 0; i < _creatorInstanceIds[creatorCore].length;) {
```

Comment:
No need to instantiate i (or j below).  Saves gas

Reply:
Done